ibm/service/cdtektonpipeline/resource_ibm_cd_tekton_pipeline.go

```diff
@@ -119,11 +120,12 @@ func ResourceIBMTektonPipeline() *schema.Resource {
 						},
 						"service_instance_id": &schema.Schema{
 							Type:        schema.TypeString,
-							Computed:    true,
+							Required:    true,
 							Description: "UUID.",
 						},
 						"id": &schema.Schema{
 							Type:        schema.TypeString,
+							Optional:    true,
```

Comment:
Will id be a optional argument?

Reply:
Actually no. The definition id would be computed upon creation and would not change. I'll remove the optional arg